lib/seed/runner.rb

```diff
@@ -8,7 +8,7 @@ class Runner
     include ActiveSupport::Benchmarkable
     include ConsoleLogger
     SIZES = Facility.facility_sizes
-    FEATURE_FLAGS_ENABLED_BY_DEFAULT = %i[drug_stocks drug_stocks_mobile appointment_reminders]
+    FEATURE_FLAGS_ENABLED_BY_DEFAULT = %i[drug_stocks drug_stocks_mobile notifications]
```

Comment:
This won't turn it on in Prod, right? I must not understand the relationship between these defaults and our prod flags.

Reply:
Correct, this is just seed data, so will only impact dev and sandbox (when we rerun seeds there).